src/Pages/List/List.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import ListTag from "./Components/ListTag";
+import ListCategory from "./Components/ListCategory";
+import PopularCreator from "./Components/PopularCreator";
+import { AiFillCaretDown } from "react-icons/ai";
+import "./List.scss";
+
+class List extends Component {
+  constructor() {
+    super();
+    this.state = {
+      listName: "사진",
+      bannerBgSrc:
+        "https://images.unsplash.com/photo-1536924940846-227afb31e2a5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1047&q=80",
+      categoryToggle: false,
+      category: [],
+      listBannerTags: [],
+      popularCreator: [],
+    };
+  }
+
+  componentDidMount() {
+    // 상단 배너 카테고리
+    fetch("http://localhost:3000/Data/List/CATEGORY.json", {
+      method: "GET",
+    })
```

Comment:
- method가 GET 일때에는 아래와 같이 생략 가능합니다. 

- fetch API도 config.js 에서 변수에 선언하여 import해 사용해주세요! 한 파일에서 관리하면 API 주소가 변동되었을 때 유지보수하기에 편리합니다!

```suggestion
    fetch("http://localhost:3000/Data/List/CATEGORY.json")
```